src/main/java/de/numcodex/feasibility_gui_backend/query/result/ResultService.java

```diff
@@ -59,6 +59,7 @@ public List<ResultLine> findSuccessfulByQuery(long queryId) {
   public void addResultLine(Long queryId, ResultLine result) {
     queryResultCache.asMap()
         .merge(queryId, QueryResult.ofResultLine(result), QueryResult::merge);
+    log.info("{};{};{}", queryId, result.siteName(), result.result());
```

Comment:
@michael-82  - should not log query results to stdout - consider using trace for this specific class